web_src/js/utils/dom.js

```diff
@@ -19,7 +19,7 @@ function assertShown(el, expectShown) {
 }
 
 function elementsCall(el, func, ...args) {
-  if (el instanceof String) {
+  if (typeof el === 'string' || el instanceof String) {
```

Comment:
Okay, I have no idea why this makes a difference…
I thought that these two calls are equivalent.
The fact that they apparently aren't is disturbing and once again proves why I hate the way JS is designed.

Reply:
That's the dark side of JS 😭